### PR TITLE
Basic test for lazy loading Case Lists

### PR DESCRIPTION
**Pull request by shubham1g5**
Tests for https://github.com/dimagi/commcare-core/pull/1388

(Skipping PR Template as there are no functional code changes)

### CODE REVIEW

File: src/test/java/org/commcare/formplayer/tests/EndpointActionsTest.kt [hunk 1]
```diff
@@ -1,23 +1,51 @@
...
+import org.springframework.context.annotation.Import
+import org.springframework.test.context.ContextConfiguration
+import org.springframework.test.web.servlet.MockMvc
 
 @WebMvcTest
-class EndpointActionsTest : BaseTestClass() {
+@Import(MenuController::class)
+@ContextConfiguration(classes = [TestContext::class, CacheConfiguration::class])
+@ExtendWith(InitializeStaticsExtension::class)
+class EndpointActionsTest {
```
Comment (shubham1g5) [inline src/test/java/org/commcare/formplayer/tests/EndpointActionsTest.kt:25]: changes here are not related to PR and I end up doing it under a wrong assumption, but keeping the changes since it's still a good change to make.

**Review by Jtang-1 — COMMENTED** [review]
Took a look and I don't see any issues, but also don't have much knowledge on this area

**Comment by shubham1g5** [comment]
Thanks @Jtang-1 . merging since these are tests only.
